test/e2e/storage/testsuites/testdriver.go

```diff
@@ -199,6 +199,8 @@ type StressTestOptions struct {
 	NumPods int
 	// Number of times to restart each Pod.
 	NumRestarts int
+	// Number of snapshots to create for each volume.
+	NumSnapshots int
```

Comment:
I'm missing the config change to enable this test for gce and hostpath csi drivers.

Reply:
Done